tests/testthat/test-lint_package.R

```diff
@@ -1,13 +1,11 @@
-context("Integration tests for `lint_package`")
-
 # When called from inside a package:
-# lint_package(".")
+# lint_package(".") # nolint
```

Comment:
Might be a good idea to # nolint this according to the new #660 syntax once we settle on a final default.

Reply:
how about a follow-up issue for tagging all `# nolint`s in the package that way?

```
./R/closed_curly_linter.R:                    message = "Closing curly-braces should always be on their own line, unless it's followed by an else.", # nolint
./R/function_left_parentheses.R:function_left_parentheses_linter <- function(source_file) { # nolint: function_left_parentheses_linter.
./R/get_source_expressions.R:    })(prev_locs == lag(next_locs)) # nolint
./R/open_curly_linter.R:                    message = "Opening curly braces should never go on their own line and should always be followed by a new line.", # nolint
./R/lint.R:        expr_lints <- flatten_lints(linters[[linter]](expr)) # nolint
./tests/testthat/test-deprecated.R:  expect_quiet_lint("'/blah/file.txt'", # nolint
./tests/testthat/test-lint_package.R:# lint_package(".") # nolint
./tests/testthat/test-lint_package.R:# lint_package(path_to_package) # nolint
./tests/testthat/test-rstudio_markers.R:  expect_equal(marker3$basePath, "test") # nolint
```

Reply:
Sounds good 👍